<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditProductWindow</class>
 <widget class="QWidget" name="EditProductWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактирование продукта</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelName">
     <property name="text">
      <string>Название</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="nameInput">
     <property name="placeholderText">
      <string>Введите название продукта</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelPrice">
     <property name="text">
      <string>Цена</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="priceInput">
     <property name="placeholderText">
      <string>Введите цену продукта</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
</ui>
